modules/@apostrophecms/ui/ui/apos/scss/global/_theme.scss

```diff
@@ -64,6 +64,7 @@
   --a-weight-bold: 700;
 
   // Type Sizes
+  --a-type-tiny: 9px;
```

Comment:
I start to wonder if we're gaining anything once it's broken down this much. That one couldn't have been 10px?

Reply:
not tiny enough

Reply:
Clearly.